src/foam/u2/view/OverlayActionListView.js

```diff
@@ -0,0 +1,182 @@
+    },
+    {
+      class: 'URL',
+      name: 'activeImageURL',
+      value: 'images/Icon_More_Active.svg'
+    },
+    {
+      class: 'URL',
+      name: 'restingImageURL',
+      value: 'images/Icon_More_Resting.svg'
+    },
+    {
+      class: 'URL',
+      name: 'hoverImageURL',
+      value: 'images/Icon_More_Hover.svg'
+    },
+    {
+      class: 'URL',
+      name: 'disabledImageURL',
+      value: 'images/Icon_More_Disabled.svg'
+    },
+    {
+      class: 'URL',
+      name: 'imageURL_',
+      expression: function(restingImageURL, hoverImageURL, disabledImageURL, activeImageURL, hovered_, disabled_, active_) {
+        if ( disabled_ ) return `url(${disabledImageURL})`;
+        if ( active_ ) return `url(${activeImageURL})`;
+        if ( hovered_ ) return `url(${hoverImageURL})`;
+        return `url(${restingImageURL})`;
+      }
+    },
+    {
+      class: 'Boolean',
+      name: 'hovered_'
+    },
+    {
+      class: 'Boolean',
+      name: 'disabled_'
+    },
+    {
+      class: 'Boolean',
+      name: 'active_'
+    },
+    {
+      class: 'FObjectProperty',
+      of: 'foam.u2.Element',
+      name: 'overlay_',
+      factory: function() {
+        return this.OverlayDropdown.create();
+      }
+    },
+    {
+      class: 'Boolean',
+      name: 'overlayInitialized_'
+    }
+  ],
+
+  css: `
+    ^action {
+      padding: 10px;
+    }
+
+    ^action[disabled] {
+      color: #aaa;
+    }
+
+    ^action:hover:not([disabled]) {
+      cursor: pointer;
+      background-color: %ACCENTCOLOR%;
+    }
+
+    ^icon {
+      background-size: 24px;
+      width: 24px;
+      height: 24px;
+    }
+
+    ^icon:hover {
+      cursor: pointer;
+    }
+  `,
+
+  methods: [
+    function initE() {
+      var self = this;
+
+      this.onDetach(this.active_$.follow(this.overlay_.opened$));
+      this.onDetach(this.disabled_$.follow(this.ExpressionSlot.create({
+        args: this.data.map((action) => action.createIsAvailable$(this.obj$)),
+        code: (...rest) => ! rest.reduce((l, r) => l || r, false)
+      })));
+
+      this.
+        start('span').
+          addClass('noselect').
+          start().
+            addClass(this.myClass('icon')).
+            style({ 'background-image': this.imageURL_$ }).
+            callIf(this.data.length > 0, function() {
+              this.
+                on('mouseover', self.onMouseOver).
+                on('mouseout', self.onMouseOut).
+                on('click', function(evt) {
+                  if ( self.disabled_ ) return;
+                  if ( ! self.overlayInitialized_ ) self.initializeOverlay();
+                  self.overlay_.open(evt.clientX, evt.clientY);
+                });
+            }).
+          end().
+        end();
+    },
+
+    function initializeOverlay() {
+      var self = this;
+      this.overlay_.forEach(this.data, function(action) {
+        this.
+          start().
+            show(action.createIsAvailable$(self.ConstantSlot.create({
```

Comment:
`self.obj$` instead of `self.ConstantSlot.create...` would be shorter to type, but it would also create a dynamic binding on self.  If self.obj isn't ever going to change then I guess this works :).